hotpdf/hotpdf.py

```diff
@@ -16,20 +14,33 @@ class HotPdf:
     def __init__(
         self,
         extraction_tolerance: int = 4,
+        source: Union[str, bytes, None] = None,
```

Comment:
this isn't updated in the docstring, is it suppose to be file?

Reply:
I need to update all the docstring